tools/project_manager/edit_common.inc

```diff
@@ -77,9 +77,9 @@ function language_list($language)
 
 function genre_list($genre)
 {
-    global $GENRES;
+    $genres = load_genre_translation_array();
```

Comment:
Perhaps this could also be translated and sorted.

Reply:
We could move the sorting-by-value into this function, sure. I don't understand your question about being translated however, can you elaborate please?

Reply:
Sorry, forget translate, wasn't thinking clearly.

Reply:
Updated the function to return the list sorted by the translated name.